### DISCUSSION TITLE
disable max_claim_radius_value

### DISCUSSION THREAD

**Discussion by lacerda401** [TownyAdvanced/Towny#5346] [Q&A] [answered]
Is there any way to disable the max_claim_radius_value to make players /town new and /town claim be only in the chunk the player current is?

**Comment by LlmDl** [comment, marked as answer]
/town new is always a single chunk.

/town claim [auto|rect|circle|#] requires the `towny.command.claim.multiple` permission node.

**Reply by lacerda401** [reply]
when I execue /town new, it takes more than 1 chunk
Im using the latest version

**Reply by lacerda401** [reply]
nvm, i was testing as an operator
